src/bitcoind.ts

```diff
@@ -1,5 +1,146 @@
+import Client from "bitcoin-core"
 import _ from "lodash"
-import { BitcoindClient, bitcoindDefaultClient, btc2sat } from "./utils"
+import { btc2sat } from "./utils"
+
+const connection_obj = {
```

Comment:
this should be constructed in the `config.ts` module and injected. To move forward we could leave it like this and perhaps @vindard could take care of it as part of his ongoing config refactoring.